VotingApplication/VotingApplication.Web.Api/Logging/LoggerFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace VotingApplication.Web.Api.Logging
+{
+    public class LoggerFactory
+    {
+        private static ILogger _logger;
+
+        public ILogger GetLogger()
+        {
+            return _logger ?? new ConsoleLogger();
```

Comment:
It appears that _logger is never set, so this always returns a new ConsoleLogger()

We should either be setting _logger with new ConsoleLogger() if null, or passing a logger type in through the constructor (which is where ninject would hopefully hook in to)